include/geode/mesh/builder/regular_grid_solid_builder.h

```diff
@@ -23,6 +23,7 @@
 
 #pragma once
 
+#include <geode/mesh/builder/coordinate_reference_system_managers_builder.h>
```

Comment:
Not sure if this include is necessary if it's the only modification in this file -> is it to avoid making breaking changes ?
If so, add it to regular_grid_surface_builder too